xapian-applications/omega/templates/query

```diff
@@ -49,17 +50,18 @@ document.write("<span title=\""+D+" "+T+"\">Modified:<br><b>"+D+"</b></span>");
 // -->
 </script>
 </head>
+
 <body bgcolor="#ffffff">
 <form name=P method=get
 action="$html{$or{$env{SCRIPT_NAME},omega}}" target="_top">
 <center>
+<fieldset>
```

Comment:
I have missed id's on a lot of sections and fieldsets. It would be wise to call them query-fields